exercises/collatz-conjecture/src/main/java/CollatzCalculator.java

```diff
@@ -0,0 +1,7 @@
+class CollatzCalculator {
+
+    int computeStepCount(final int start) {
```

Comment:
I would remove the `final` keyword here. It's fine to keep it in the reference solution but I think it might be a bit confusing to people who are new to Java :) What do you think?

Reply:
That makes sense to me - I'll get it done when I'm back at my computer!